docs/architecture/microservices/toc.yml

```diff
@@ -25,7 +25,7 @@
       href: net-core-net-framework-containers/net-container-os-targets.md
     - name: Official .NET Docker images
       href: net-core-net-framework-containers/official-net-docker-images.md
-  - name: Architecting Container and Microservice Based Applications
+  - name: Architect Container and Microservice-Based Apps
```

Comment:
Without the -ing, the word "architect" isn't readily recognized as a verb without more context.  And a hyphen before Microservice would require a hyphen before Container since the intended meaning is container-based and microservice-based.